tailwind.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
- will we have all our components & app folder in a common folder like `src` if so can we just add the path to that common folder?

Reply:
So currently, this is how our folder structure is, that's why didn't change this.